### ISSUE TITLE
404 Error on NBA Height Example

### ISSUE THREAD

**Issue by henry232323** [roclark/sportsreference#522]
**Describe the bug**
Copied directly from the documentation [here](https://sportsreference.readthedocs.io/en/stable/examples.html), the code below gives a 404 error, instead of yielding any proper output.

**To Reproduce**
Sample code which causes an issue.
```python
from sportsreference.nba.teams import Teams

def get_height_in_inches(height):
    feet, inches = height.split('-')
    return int(feet) * 12 + int(inches)

def print_tallest_player(team_heights):
    tallest_player = max(team_heights, key=team_heights.get)
    print('%s: %s in.' % (tallest_player, team_heights[tallest_player]))

for team in Teams():
    print('=' * 80)
    print(team.name)
    team_heights = {}
    for player in team.roster.players:
        height = get_height_in_inches(player.height)
        team_heights[player.name] = height
    print_tallest_player(team_heights)
```

**Traceback**
```python
Traceback (most recent call last):
  File "<pyshell#6>", line 1, in <module>
    for team in Teams():
  File "C:\Users\henry\AppData\Local\Programs\Python\Python38-32\lib\site-packages\sportsreference\nba\teams.py", line 591, in __init__
    self._retrieve_all_teams(year)
  File "C:\Users\henry\AppData\Local\Programs\Python\Python38-32\lib\site-packages\sportsreference\nba\teams.py", line 718, in _retrieve_all_teams
    doc = pq(SEASON_PAGE_URL % year)
  File "C:\Users\henry\AppData\Local\Programs\Python\Python38-32\lib\site-packages\pyquery\pyquery.py", line 224, in __init__
    html = url_opener(url, kwargs)
  File "C:\Users\henry\AppData\Local\Programs\Python\Python38-32\lib\site-packages\pyquery\openers.py", line 84, in url_opener
    return _requests(url, kwargs)
  File "C:\Users\henry\AppData\Local\Programs\Python\Python38-32\lib\site-packages\pyquery\openers.py", line 68, in _requests
    raise HTTPError(resp.url, resp.status_code,
urllib.error.HTTPError: HTTP Error 404: Not Found
```

**Desktop (please complete the following information):**
 - Produced on both Ubuntu 20.04.1 and Windows 10
 - Sportsreference Version 2.23.0

**Comment by roclark** [comment]
Hey @henry232323, thanks for creating this issue! I believe this issue has been resolved in the upstream code and is related to #481. I was able to take the code on my end with the latest upstream and have it run successfully. Are you able to use the code directly from source? If not, I am hoping to release the next version fairly soon in which you can update then. Thanks again!

**Comment by henry232323** [comment]
The example code does seem to work fine on the latest. Thank you!
